atsign-foundation/at_app_cli
# Build test for at_app

**- What I did**

Review and merge #85 first
Run the existing build scripts as a GitHub action

**- How I did it**

Local changes

**- How to verify it**

I know the build scripts work, but I'm new to writing workflows.
Please verify that it's setup correctly and that the appropriate terms are used.

**- Description for the changelog**
Build test for at_app
